src/Wrapper/CheapestVariation.php

```diff
@@ -91,4 +91,18 @@ private function setDefaultPrice(array $prices): void
 
         $this->item->addPrice(min($prices));
     }
+
+    /**
+     * @param array<string,string> $cheapestVariationsData
```

Comment:
Shouldn't this be `array<string, string|float>`, because the array also contains the price of the variation?
```suggestion
     * @param array<string, string|float> $cheapestVariationsData
```